sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -17,9 +17,9 @@
 import com.azure.core.http.policy.HttpPolicyProviders;
```

Comment:
We don't want to do the `endpoint` to `vaultUrl` change for Certificates client, is it?

Reply:
Certificate changes not part of this PR. separate PR.